src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -510,56 +474,41 @@ def generate_maps(
     return maps
 
 
-def subset_results_by_state(processed_results_dir: str, state: str) -> None:
+def subset_results_by_state(processed_results_dir: Path, state: str) -> None:
     # Loads final results and subsets those files to a provide state excluding the Social Security Observer
 
     abbrev_name_dict = {v: k for k, v in metadata.US_STATE_ABBRV_MAP.items()}
     state_name = sanitize_location(abbrev_name_dict[state.upper()])
-    processed_results_dir = Path(processed_results_dir)
-    directories = [x for x in processed_results_dir.iterdir() if x.is_dir()]
-    # Get USA results directory
-    for directory in directories:
-        if "pseudopeople_input_data_usa" in directory.name:
-            usa_results_dir = directory
-    # Parse version number if necessary
-    version = usa_results_dir.name.split("_")[-1]
-    if version != "usa":
-        state_dir = (
-            processed_results_dir
-            / "states"
-            / f"pseudopeople_input_data_{state_name}_{version}"
-        )
-    else:
-        state_dir = processed_results_dir / "states" / f"pseudopeople_input_data_{state_name}"
+    version = processed_results_dir.name.split("_")[-1]
```

Comment:
since we handled the `processed_results_dir` up front in cli.py, this all is simpler